### ISSUE TITLE
Raw metadata always

### ISSUE THREAD

**Issue by InTEGr8or** [prose/prose#1072]
I thought my issue might already be handled so I filtered the issues by "metadata" and saw that 150 out of 159 issues are metadata-related, so I think this might be useful to some people.

I want raw metadata always.

I looked in the config section and I see the baffling number of changes I can make to the metadata, but I don't think it will work because right now, without config set on title, the title is supposed to display.

I am using Hugo for rendering and using title in the _index.md of every folder, and in every post. Somewhat bazzarly, this element is hidden from my view everywhere I look in Prose. This is a show stopper for me, in what looks like an otherwise perfect solution for my needs.

Is there a configuration setting for "always display all metadata in raw markdown just exactly like it's written"? Or would that require some code change? I wasn't able to find this option in the _prose.yml settings descriptions or by experimenting with the _prose.yml.

The documentation appears to say that if a setting is not set the default is to display the field, but **title** is not shown, and **description** is hidden inside the **metadata** button, even though I have not included it in the _prose.yml.

I am using: Chrome browser.
I tried a variety of settings such as 
```prose:
  metadata: 
    title: true```
and 
```prose:
  metadata: 
    field:
      element: "text"
      name: "title"```

**Comment by dereklieu** [comment]
If you exclude any metadata configuration, then the title field just becomes a specialized field for the `title` property. 

<img width="569" alt="screen shot 2018-04-16 at 11 46 39 am" src="https://user-images.githubusercontent.com/1590802/38820418-d7c21a6e-416b-11e8-9cdf-9eea8ab7b911.png">

*Changing the value just changes the `title` metadata*

## Using the raw editor to add change the title

You can also set the title from the raw editor, although you're right that Prose will try to hide this from you on refresh.

![title](https://user-images.githubusercontent.com/1590802/38820444-ee19d09a-416b-11e8-9c0f-147f7a7d9821.gif)

I'm having trouble understanding your issue, since regardless of how you do it, it all gets put into the same `title` tag. Is there some reason you need this to actually appear in the raw editor?

**Comment by InTEGr8or** [comment]
Maybe I am missing something. When I view this file on Github, I see a title inside yaml fencing.
![image](https://user-images.githubusercontent.com/1802850/38845708-a1883766-41ad-11e8-96a4-c0645fde1e9e.png)

When I view the same file in Prose.io, I don't see the title in the metadata. 
![image](https://user-images.githubusercontent.com/1802850/38845908-836c6648-41ae-11e8-8be9-2a2866333b98.png)

I would really like some way to just show the metadata in the Markdown. In my use case hiding it is a unnecessary complication.

Thanks for the quick response, btw.

I see that I can change the title in text next to the M↓. I can use that for now.

**Comment by dereklieu** [comment]
Yeah I totally understand the weirdness here around hiding the title. For reference, this behavior should only apply to markdown files, and (making some assumptions here) is likely due to Prose being developed with Jekyll in mind, where a markdown file is generally expected to have a `title` metadata property that is preceded by a specifically formatted date. This could be annoying if you just want a general `title` field and have another processor behind Jekyll in mind. 

I'd be open to ideas here on how to make the `title` field generalizable if you're not using Jekyll. One annoying thing I can see now is that it's currently impossible to specify a `title` frontmatter property in a markdown file, and not have the markdown file's filename be preceded by the date.
